content/en/docs/languages/go/quickstart.md

```diff
@@ -25,6 +25,15 @@ spelling: cSpell:ignore Fatalf GOPATH
       $ go get google.golang.org/protobuf/cmd/protoc-gen-go \
                google.golang.org/grpc/cmd/protoc-gen-go-grpc
       ```
+      
+      If you're using Go v1.16 or later:
+ 
+      ```sh
+      $ go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+      $ go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
```

Comment:
Maybe we should prescribe a version # here (major+minor, not patch), so it doesn't change at all over time?

Reply:
You are correct. I'll change it as follows
```
	$ go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.26.0
	$ go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1.0
```

Reply:
@uta-mori: please drop the patch numbers:

- v1.26.0 --> v1.26
- v1.1.0 --> v1.1

Reply:
I've dropped the patch numbers.